packages/@expo/metro-config/src/serializer/serializeChunks.ts

```diff
@@ -513,11 +514,14 @@ export class Chunk {
         // TODO: Make this user-configurable.
 
         // Make all paths relative to the server root to prevent the entire user filesystem from being exposed.
-        if (module.path.startsWith('/')) {
+        if (path.isAbsolute(module.path)) {
```

Comment:
This PR should have been backported already, not sure why it wasn't. Could have been overwritten somehow

Reply:
nvm, that was this PR https://github.com/expo/expo/pull/33503 - wasn't backported yet